utils/brand.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+GIT_BRANCH="$(git branch | grep '\*' | cut -d' ' -f2-)"
+GIT_REMOTE="$(git config "branch.${GIT_BRANCH}.remote")"
+GIT_URL="$(git config --get "remote.${GIT_REMOTE}.url")"
+if [[ "${GIT_URL}" == git@* ]]; then
+    GIT_URL="${GIT_URL/://}"
+    GIT_URL="${GIT_URL/git@/https://}"
```

Comment:
This following part is not back ported, is on purpose ?

https://github.com/searxng/searxng/blob/3dea7e609bac72d4a129979ba15315b966ff020f/searx/version.py#L55-L56

The purpose is to remove the end part of `git@github.com:searxng/searxng.git`.

Reply:
Thanks for having an eye on .. [fixed it](https://github.com/searxng/searxng/compare/fc5c0060942cba4408efe71a2ff8d51ff75eed72..c881f9f0e1fe9b10733e3097ee4f87e3d9dc07e0)